.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,47 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '0 */12 * * *'
```

Comment:
Could you add a comment for how often this is? I assume every 12 hours?

Reply:
Yes, added a comment